narwhals/_duckdb/expr.py

```diff
@@ -33,7 +33,7 @@
     from narwhals.utils import Version
 
 
-class DuckDBExpr(CompliantExpr["duckdb.Expression"]):
+class DuckDBExpr(CompliantExpr["duckdb.Expression"]):  # type: ignore[type-var]
```

Comment:
This ignore should actually be solved. `duckdb.Expression` is not a `CompatibleSeries`

Is it ok for a follow-up?

Reply:
yup definitely